pyccel/codegen/printing/pycode.py

```diff
@@ -322,6 +322,7 @@ def _print_Len(self, expr):
         return 'len({})'.format(self._print(expr.arg))
 
     def _print_Import(self, expr):
+        print(expr)
```

Comment:
Forgotten `print`. You will also need to add `min`/`max` to [this dict](https://github.com/pyccel/pyccel/blob/master/pyccel/codegen/printing/pycode.py#L38-L46) to ensure that the `amin` is imported when translating things like:
```
from numpy import max
x = max(y)
```